src/bin/cargo/commands/help.rs

```diff
@@ -46,8 +46,16 @@ fn try_help(config: &Config) -> CargoResult<bool> {
         Some(s) => s,
         None => return Ok(false),
     };
-    // Check if this is a built-in command (or alias);
-    let subcommand = match check_alias(config, subcommand) {
+
+    // Check if this is an alias. If so, just display the alias information.
+    if let Some(argv) = check_alias(config, subcommand) {
+        let alias = argv.join(" ");
+        drop_println!(config, "'{}' is aliased to '{}'", subcommand, alias);
```

Comment:
Usually error messages in Cargo use backticks to wrap a value, but I am not sure about this kind of help message.

```suggestion
        drop_println!(config, "`{}` is aliased to `{}`", subcommand, alias);
```

Reply:
Hah yeah, I tried it both ways, and thought this one looked a little better. I didn't notice any style guidance about it. I'll see what @alexcrichton says, but easy to change either way.